tests/api.suite.dist.yml

```diff
@@ -5,3 +5,8 @@ modules:
       - REST:
           depends: PhpBrowser
           url: http://localhost/tests/joomla-cms3/
+      - SOAP:
```

Comment:
I guess you're doing this as a first test, but shouldn't those environments be defined so the multi-soap test environment is ready?